src/Controller/V2/Project/ProjectRoute.ts

```diff
@@ -0,0 +1,312 @@
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.updateProjectHandler(req, res)
+        }, next)
+      }
+    )
+
+    router.get(
+      [`${this.basePath}/:projectID`, `${this.basePath}/:projectID/manuscript/:manuscriptID?`],
+      celebrate(loadProjectSchema, {}),
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.getProjectModelsHandler(req, res)
+        }, next)
+      }
+    )
+
+    router.post(
+      `${this.basePath}/:projectID/users`,
+      celebrate(addUserSchema, {}),
+      AuthStrategy.JsonHeadersValidation,
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.updateUserRoleHandler(req, res)
+        }, next)
+      }
+    )
+
+    router.post(
+      `${this.basePath}/:projectID/manuscript/:manuscriptID?`,
+      celebrate(createManuscriptSchema, {}),
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.createManuscriptHandler(req, res)
+        }, next)
+      }
+    )
+
+    router.get(
+      `${this.basePath}/:projectID/collaborators`,
+      celebrate(projectCollaboratorsSchema),
+      AuthStrategy.JsonHeadersValidation,
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.getCollaboratorsHandler(req, res)
+        }, next)
+      }
+    )
+
+    router.get(
+      [
+        `${this.basePath}/:projectID/archive`,
+        `${this.basePath}/:projectID/manuscript/:manuscriptID/archive`,
+      ],
+      celebrate(getArchiveSchema, {}),
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.getArchiveHandler(req, res)
+        }, next)
+      }
+    )
+
+    router.get(
+      `${this.basePath}/:projectID/:scope`,
+      celebrate(accessTokenSchema, {}),
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.generateAccessTokenHandler(req, res)
+        }, next)
+      }
+    )
+
+    router.delete(
+      `${this.basePath}/:projectID`,
+      celebrate(deleteSchema, {}),
+      AuthStrategy.JsonHeadersValidation,
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.deleteProjectHandler(req, res)
+        }, next)
+      }
+    )
+  }
+  private async createProjectHandler(req: Request, res: Response) {
```

Comment:
I don't like the `Handler` suffix for these functions. Can we remove it or would there be a conflict?